CHANGELOG.md

```diff
@@ -33,6 +33,12 @@
   ```
   
   In Dart SDK `>=1.25.0` this can be relaxed as `part of` can refer to a path.
+  To opt-in, `GeneratorBuilder` now has a new flag, `requireLibraryDirective`.
```

Comment:
Also I forgot about the fact that you need to update your sdk constraint if you are publishing these files as part of a package, so using it automatically is a no-go.

What we could do though is just add the sdk constraint in this package?